inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -182,6 +182,11 @@ int main(int argc, char *argv[]) {
             slog::info << "GPU extensions is loaded " << ext << slog::endl;
         }
 
+        // TODO May need an update
+        if (FLAGS_d.find("PlaidML") != std::string::npos) {
```

Comment:
Didn't we allcaps this?

Reply:
This isn't needed at all in 2021.1 and I overlooked getting rid of it. Fixed.